backend/src/guided-development.ts

```diff
@@ -51,13 +51,33 @@ export class GuidedDevelopment {
             if (subItem.fqid === itemFqid) {
               return subItem;
             }  
+            else if( subItem.items ){
+              const deepSubitem =  this.getDeepSubitem(itemFqid, subItem);
```

Comment:
you can immediately return the result of this.getDeepSubitem(itemFqid, subItem)
No need to use if , in case it is undefined the returned value is also undefined

Reply:
If I do that then it will not check for the next item and its subitems.

Reply:
ok